### ISSUE TITLE
Research users priorities

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeREGNonProductIssues#28]
What are our researchers' priorities
- Should we try to complete those counties where there are a lot already.
- Should we concentrate on pre-1837
- Should we try to fill in gaps.
- Should we start all counties from 1800 to 1837.
NB current priority to get counties to 'tipping point' of usefulness - where researchers finding it useful, become transcribers.

**Comment by Sherlock21** [comment]
An interesting question.

How do you define such Counties? - i.e. what constitutes :"Finished' - up to 1917 ( under the So Called year 100 year rule)?
Because 1837 was not mandatory, many still only did the old Baptism until about 1870 or so. so do 1800 to 1880 in this case. So I would say : do up to 1880 on the first pass if there are other records for that Placename.

More important top "complete" a PLACE rather than a County, in my view, whichever way you define 'Complete'. A Second pass would be to Complete a County.

What do you define as gaps? whole church sets not done? Like C of E Parish Church done but not Non Conformists not the RC church if they exists? or do you mean a few years missing from the Parish Church - OR all these.

What do you define as this "Tipping Point " please? and is there evidence to support the suggestion that this would increase the number of Transcribers.?

And do you have enough basic PR images to give these people the work to do? CC's tell me that many of the existing Transcribers have NO work now.

**Comment by Captainkirkdawson** [comment]
I agree fully with EricB when he calls this an interesting question.

If we had freedom of choice then answering it could be extemely useful. 

But as he notes and as we have repeatedly stated over the past 20 years. We cannont get the data to transcribe.  Henc in most cases we simply do what data to which we have access.

If the question was to inform a systematic  push to obtain access to data then an answer would be useful. 

But my views on this have been ignored for the past 20 years and continue to be ignored.

**Comment by SteveBiggs** [comment]
For counties where we do have comprehensive data such as Sussex, my view would be to start with pre-1837 since the GRO records comprehensively transcribed by FreeBMD start from then. This is also the mission statement of FreeREG.

I entirely agree with Kirk and EricB that we need to sort out the problem of access to data for many counties.  Many CC's have approached their local RO with varying success and we need a more formal central approach backed up by legal advice on rights of access to the data. Pat, Mick Claxton and I are trying to fix a meeting to progress this.

There are a few counties where we have extensive data and if there are transcribers with nothing to do, we should temporarily assign them to work in these counties until data is obtained for their preferred one.

**Comment by Captainkirkdawson** [comment]
Steve

As FreeREG's data manager for nearly 10 years and the initial liaison with the Latter Day Saints in Utah I would really like to be involved in any such discussion.

**Comment by SteveBiggs** [comment]
Kirk, I would be very happy to have you involved in the discussion.

**Comment by edickens** [comment]
The "NB Current tipping point", which I see as 250K records per county, is my suggestion.  It is not based on any scientific fact.  Just that when I was CC for SOM, that was when things took off.  But counties are of very different sizes, so this principle would not apply to every county.
I do not have any reason for going for any of the suggestions.  It is now just where we can get images.
What perhaps is of more interest to a researcher would be information on when we know that we have competed a Church up to 1837, or 1870, or 1917.

**Comment by Sherlock21** [comment]
I agree with Eric D's last para above. 
Like: change the colour of the Place /Church name in the view that Researchers get to see perhaps ? but minimum effort on the part of coders working out how to do it!

**Comment by Sherlock21** [comment]
Would it be useful to consider counting how many times a County/Placename is used in a Search Specification by the researcher, as a measure of whats used is most popular, so put effort into completing these Placenames and their Nearby neighbours.

Count per week perhaps? or some other fixed period?
This might be concluded that these are the Places most needed but the researcher.
